### ISSUE TITLE
Silhouette sign

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6711]
**Preferred term label:**
Silhouette sign

**Synonyms**

**Definition (free text, please give PubMed ID)**

The silhouette sign is the absence of depiction of an anatomic soft-tissue border. It is caused by consolidation and/or atelectasis of the adjacent lung, by a large mass, or by contiguous pleural fluid. The silhouette sign results from the juxtaposition of structures of similar radiographic attenuation. The sign actually refers to the absence of a silhouette. 

PMID:18195376

Comment
The silhouette sign is not always indicative of disease (eg, unexplained absence of right border of heart is seen in association with pectus excavatum and sometimes in healthy individuals). See Figure 67 of Hansell DM, et al., Fleischner Society: glossary of terms for thoracic imaging. Radiology. 2008 Mar;246(3):697-722.

Parent term (use hpo.jax.org/app)
Abnormal pulmonary thoracic imaging finding HP:0031983

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
